changelog.txt

```diff
@@ -1,7 +1,18 @@
 *** Facebook for WooCommerce Changelog ***
 
-2021-xx-xx - version 2.5.0
- *  Ensure variable product attribute values containing , sync correctly to Facebook additional_variant_attribute field.
+2021-xx-xx - version 2.5.0-rc1

```

Comment:
I'd probable leave the changelog version as 2.5.0 here. I doubt Woorelease will bump this version automatically so it's one less manual thing to do.

Reply:
Yep good point - thanks :)

Reply:
WooRelease supports `--product_version` that allows you to specify the version of the plugin you want to release. There are some configuration steps needed but this is a one time effort.
https://github.com/woocommerce/woorelease/blob/develop/README.md#steps-to-release-a-product